pkg/platformconfig/platformconfig.go

```diff
@@ -186,6 +190,52 @@ func (c *Config) GetFunctionMetricSinks() (map[string]MetricSink, error) {
 	return c.getMetricSinks(c.Metrics.Functions)
 }
 
+func (c *Config) EnrichContainerResources(ctx context.Context,
```

Comment:
add some comments what/why you are enriching and the logic. describe it above the function signature, use go convention for function documenting